WordPress/Classes/ViewRelated/Reader/ReaderStreamViewController.swift

```diff
@@ -1579,10 +1579,13 @@ extension ReaderStreamViewController: NoResultsViewControllerDelegate {
 
         let buttonTitle = buttonTitleForTopic(topic)
```

Comment:
Rather than fetching the button title, can you just check `if ReaderHelpers.topicIsFollowing(topic)` and `if ReaderHelpers.topicIsLiked(topic)` directly?

Reply:
🤦‍♂️ . So obvious. Yes, of course.